camera_pos_utils.py

```diff
@@ -138,5 +138,5 @@ def translate4(x, y, z):
                       [-0.68636268, 0.42995355, -0.58655453]])
     T_vec = np.array([-0.32326042, -3.65895232, 2.27446875])
     init_pose = compose_44(R_mat, T_vec)
-    imagelocs = Images_Meta("/home/cviss/PycharmProjects/GS_Stream/data/UW_tower/sparse/0/images.txt")
+    imagelocs = ImagesMeta("/home/cviss/PycharmProjects/GS_Stream/data/UW_tower/sparse/0/images.txt")
```

Comment:
Just out of curiosity, what is images.txt? What's the role of this file?

Reply:
As far as I know, class names in Python should follow the CamelCase convention. I think changing it was a good move. 👍

Reply:
Thanks! Yeah I was super sloppy with the naming conventions... The images.txt is from colmap and it contains the image name and camera poses (as quaternion). This is necessary for me to get the closest images to the current pose.